libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -34,6 +35,11 @@ export class BookSearchComponent implements OnInit {
 
   ngOnInit(): void {
     this.books$ = this.store.select(getAllBooks);
+    this.searchForm.controls.term.valueChanges.pipe(debounceTime(500),
```

Comment:
Nice use of `valueChanges`, `debounceTime`, and `distinctUntilChanged` here. Make sure you clean up your subscription here when the component is destroyed to stop memory leaks.